usr/share/gamescope-session/sessions.d/steam-plus

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+STEAM_SESSION_FILE=/usr/share/gamescope-session/sessions.d/steam
+[[ -f "${STEAM_SESSION_FILE}" ]] && . "${STEAM_SESSION_FILE}"
+
+CURSOR_FILE="${HOME}/.local/share/Steam/tenfoot/resource/images/cursors/arrow_right.png"
+CLIENTCMD="opengamepadui --only-qam -- $CLIENTCMD"
```

Comment:
Is the intention here to concatenate the historical STEAMCMD and OGUICMD? The second half of this needs to be the command for the underlay process that OpenGamepadUI will launch. This is typically steam and all of its arguments. Previously we separated this into two variables. If you echo $CLIENTCMD here will it be "opengamepadui --only-qam -- steam -steamos3 -etc"?

Reply:
Also, we're going to change the name from --only-qam so we should update that now and merge everything at the same time.

Reply:
Yes, this would source the whole steam gamescope session, and execute the `CLIENTCMD` that is defined in the steam session.

Reply:
Yes, it works like ShadowApex explained it. Is that acceptable?

Reply:
This was already the case before. I think we can just make it an optional dependency. If steam/steam session is not available, it will currently just fail. I am not sure what alternative behaviour we could have.

Reply:
> This was already the case before. I think we can just make it an optional dependency. If steam/steam session is not available, the steam-plus session will currently just fail. I am not sure what alternative behaviour we could have.

Not exactly. Before it was able to pick up a custom STEAMCMD from a .config/environment.d conf, but the default STEAMCMD was explicit in the session script.

We probably don't want to pull from the steam session script anyway as it endures we will always need the same steam arguments for both sessions, which we're working towards removing as many as possible in the Plus session.

Reply:
Can we adjust this?